File: Domain/Entities/Answer.cs
namespace Domain.Entities;

public class Answer
{
    public int Id { get; set; }
    
    public string Content { get; set; }
    
    public int QuestionId { get; set; }
    public Question Question { get; set; }
    
    public Guid Customer { get; set; }
    public Order Order { get; set; }
}

File: WebFront/Api/QuizSolve/Answer.ts
import { QuestionDto } from './QuestionDto';

export class Answer {
    public Content: string[];
    public Question: QuestionDto;
    
    constructor(question: QuestionDto) {
        this.Question = question;
    }
    
    public withContent(content: string[]): Answer {
        this.Content = content;
        return this;
    }
}